### ISSUE TITLE
Collections auto-refresh on start

### ISSUE THREAD

**Issue by bumi** [bumi/tasveer#16]
Currently it seems the only option to get new photos in the collection is to edit/save the collection again. 

The collection should be updated everytime the application is opened. Newly found photos should be added to the collection and uploaded.

**Comment by bumi** [comment]
related PR: https://github.com/bumi/tasveer/pull/32

can we run this periodically? right now if the app is not closed by the user but only sits in the background photos are also not synchronized.

**Comment by bumi** [comment]
I think the better option is to run this whenever the app gets focus (on an UIApplicationDidBecomeActiveNotification?)

**Comment by bumi** [comment]
@ludoded any updates on this? what do you think? would love to finish this topic.

**Comment by ludoded** [comment]
@bumi Yes I'll work on this tomorrow. I'll implement trigger that will start the operation that I start on launch of the app, but will observe it on UIApplicationDidBecomeActiveNotification.
UIApplicationDidBecomeActiveNotification is also posted when app is initially launched, so I basically can remove the code that I wrote for launch fetching new photos and implement it only for this notification handler.

**Comment by bumi** [comment]
ok, cool. 
yeah, should be the same code just the operations should be triggered from that notification, right?

**Comment by bumi** [comment]
please work on this first thing... this is still the main blocker to test anything.

**Comment by ludoded** [comment]
Sure, I’ll work on this tomorrow. I’m flying back to home today and will be at home tomorrow at noon local time
